doc/modules/preprocessing.rst

```diff
@@ -423,3 +423,24 @@ values than observed values.
 
 :class:`Imputer` can be used in a Pipeline as a way to build a composite
 estimator that supports imputation. See :ref:`example_missing_values.py`
+
+
+Custom Transformers
+===================
+
+Often, you will want to convert an existing python function into a transformer
+to assist in data cleaning or processing. Users may implement a transformer from
```

Comment:
Whoops that is what I meant. It is useful for pipelines. I don't think searching over the function is the main use case. I think selecting columns / slicing etc is a good use case.